patient-care/webapp/client/shared/editSampleGroup.js

```diff
@@ -335,8 +336,10 @@ Template.formValuesFilter.onCreated(function(){
   instance.editing = new ReactiveVar(false);
 
   let dataset_id = instance.data.data_set_id ;
-  instance.available_filter_forms = new ReactiveVar(); 
+  instance.available_filter_forms = new ReactiveVar();
   instance.available_filter_forms.set([{name: "Loading forms...", formId: "placeholder_loadingforms"}]);
+  instance.filter_forms_options = new ReactiveVar();
```

Comment:
I did a spot-check for the code changes around it and all of them were carried over, which is why I didn't check every change. 

I just went through line-by-line for that diff. I found one other change that got lost. (https://github.com/UCSC-MedBook/MedBook_/commit/ce1b61f)

Should we go through the whole PR? There's 1926 files change... Or is does that diff cover all the important non-`node_modules` changes?